spec/features/directors/show_spec.rb

```diff
@@ -14,6 +14,7 @@
       it 'should visit the page at /director/:id' do
         visit "/directors/#{director_1.id}"
 
+        save_and_open_page
```

Comment:
Should probably take the `save_and_open_page` out before merging.

Reply:
Resolved.